skypy/halos/tests/test_colossus.py

```diff
@@ -32,3 +32,76 @@ def test_colossus_mass_sampler():
     D, p = kstest(sample, lambda t: np.interp(t, m, CDF))
 
     assert p > 0.01, 'D = {}, p = {}'.format(D, p)
+
+
+@pytest.mark.skipif(not HAS_COLOSSUS, reason='test requires colossus')
+@pytest.mark.flaky
+def test_colossus_mf_redshift():
+
+    from skypy.halos.redshift import colossus_mf_redshift
+    from astropy.cosmology import Planck18
+    from astropy import units
+    from scipy import integrate
+    from colossus.lss.mass_function import massFunction
+
+    # Parameters
+    redshift = np.linspace(0., 2., 100)
+    model, mdef = 'despali16', '500c'
+    m_min, m_max = 1e+12, 1e+16
+    sky_area = 1.0 * units.deg**2
+    cosmology = Planck18
+    sigma8, ns = 0.82, 0.96
+
+    # Sample redshifts
+    z_halo = colossus_mf_redshift(redshift, model, mdef, m_min, m_max, sky_area,
+                                  cosmology, sigma8, ns, noise=False)
```

Comment:
I get this warning, using Planck15 instead. Is this a problem?
`WARNING: Astropy cosmology class contains massive neutrinos, which are not taken into account in Colossus.`

Reply:
This is a colossus warning. Their power spectrum implementation doesn't account for massive neutrinos. It's not something we have any control over.